xml/System.Windows.Forms/TextFormatFlags.xml

```diff
@@ -171,6 +176,15 @@
       <MemberValue>0</MemberValue>
       <Docs>
         <summary>Adds padding to the bounding rectangle to accommodate overhanging glyphs.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is the default.
```

Comment:
Since remarks do not show up in intellisense in VS, This should be mentioned in the summary instead.

Reply:
I don't think remarks for enum fields show up even in docs.